### DISCUSSION TITLE
Problem on sendig user's activation link

### DISCUSSION THREAD

**Discussion by pow3r77** [globaleaks/globaleaks-whistleblowing-software#3962] [Q&A]
Hello to all,
I'm facing a problem on my firt installation fo GL system
I Have used a Ubuntu dedicated PC and alla the system seems to work well.
The smtp are correctly configured and I can receive the test email.
But when I create a new user and try to send the activation link, the users receive nothing.
I have try different user's mail domani, gmail.com, yahoo.com but nothing

Anyone can hekp me?

**Comment by maelaborazioni** [comment]
I have the same problem, _probably_ after the last update...

**Reply by maelaborazioni** [reply]
I made some test, the problem is not related to the SMTP settings. You can use whatever (working) account you want and contact any domain: the test message comes without any problem while the activation message and the new report notification do not go out.

**Comment by maelaborazioni** [comment]
We finally found a solution: if you restart your server, then the email functionality will be restored. 
Please note that you can encounter an unwanted behaviour: the old emails will be sent all togheter...
@pow3r77
